### PR TITLE
UI fixes and improvements

### PR DESCRIPTION
**Pull request by Viscount88**

### CODE REVIEW

File: fearless/Modules/Staking/StakingBondMore/StakingBondMoreViewController.swift [hunk 1]
```diff
@@ -43,7 +43,6 @@ final class StakingBondMoreViewController: UIViewController, ViewHolder {
     }
 
     private func setupAmountInputView() {
-        rootView.amountInputView.textField.keyboardType = .decimalPad
```
Comment (bnsports) [inline fearless/Modules/Staking/StakingBondMore/StakingBondMoreViewController.swift:46]: why?

File: fearless/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -810,7 +810,7 @@ Remember to make a backup of your key and keep it in a safe and private place (e
 "network.issue.network.unavailable" = "Network is unavailable";
 "network.issue.node.unavailable" = "Node is unavailable";
 "search.view.title" = "Send to";
-"search.textfield.placeholder" = "Public address";
+"search.textfield.placeholder" = " Public address";
```
Comment (bnsports) [inline fearless/en.lproj/Localizable.strings:813]: why?

File: fearless/ru.lproj/Localizable.strings [hunk 3]
```diff
@@ -807,7 +807,7 @@ and depending on the network state, may not be selected to participate";
 "network.issue.network.unavailable" = "Сеть не доступна";
 "network.issue.node.unavailable" = "Нода не доступна";
 "search.view.title" = "Получатель";
-"search.textfield.placeholder" = "Публичный адрес";
+"search.textfield.placeholder" = " Публичный адрес";
```
Comment (bnsports) [inline fearless/ru.lproj/Localizable.strings:810]: why?

**Review by bnsports — CHANGES_REQUESTED** [review]

File: fearless/Common/View/DetailsTriangularedView.swift [hunk 4]
```diff
@@ -71,7 +71,7 @@ class DetailsTriangularedView: BackgroundedContentControl {
                 }
                 if actionButton == nil {
                     let actionButton = TriangularedButton()
-                    actionButton.applyDefaultStyle()
```
[the remark this answers is not in the document]
Comment (Viscount88) [reply fearless/Common/View/DetailsTriangularedView.swift:74]: Yes, we had before two different styles, but for now amount input is same for all new screens. It's already configured inside new view.

File: fearless/Modules/Staking/StakingBondMore/StakingBondMoreViewController.swift (hunk 1 above)
Comment (Viscount88) [reply fearless/Modules/Staking/StakingBondMore/StakingBondMoreViewController.swift:46]: It always decimal pad and configured inside view

**Review by bnsports — APPROVED** [review]

**Pull request merged**